### ISSUE TITLE
feature: select instance from context menu

### ISSUE THREAD

**Issue by atomGit** [jankkm/invidioucious#1]
thanks for this extension! the ability to switch instances is a big plus, however i'd like to suggest being able to do so from the toolbar icon context menu, reason being that some instances become temporarily overloaded or off-line and it would be nice to switch quickly in such cases

**Comment by jankkm** [comment]
I will look into it. If making a popup window is not too much work I will add it.

**Comment by atomGit** [comment]
> mozilla really screwed up about:addons so maybe i'm missing something here, but i have .9.6.1 installed and i'm not seeing any menu from the toolbar icon to select the instance (R or L click)

> This issue is about the „Watch on YouTube“ link. Please check if this works for you now.

sorry, i commented in the wrong issue

i checked for add-on updates and there were none - i have 0.9.6.1 installed and there's no item in the context menu for selecting an instance

**Comment by atomGit** [comment]
as an update to this request - don't know if it's possible, but rather than creating a context menu to select an instance, could the mouse wheel be used to do this by hovering the cursor over the toolbar icon?

**Comment by jankkm** [comment]
> as an update to this request - don't know if it's possible, but rather than creating a context menu to select an instance, could the mouse wheel be used to do this by hovering the cursor over the toolbar icon?

I don't think this is possible but it is still on my todo list to add a feature that makes switching faster. I don't have a lot of time at the moment, sorry!

**Comment by atomGit** [comment]
no problem - my primary reason for wanting this has to do with non-responsive instances, which happens occasionally, or ones with a heavy load that take a loooooong time to load, like invidio.us often does ... so what i'm saying is that if #5 is addressed, then there's possibly no need to incorporate this
